### DISCUSSION TITLE
problem with custom cards persistance

### DISCUSSION THREAD

**Discussion by jwoeckinger** [Zuehlke/poinz#277] [Q&A] [answered]
Hi, i'm pretty impressed by poinz and would love to use it. Thanks for the great work & sharing it! The only thing that doesnt work as expected for me is  persisting custom cards. No matter if i do it in visual- or text-editor, when i click "i know what i'm doing" it resets to default. Tried with FF and Edge. What am i missing?

**Comment by xeronimus** [comment, marked as answer]
Hey there, thanks very much for reaching out.

could you check whether Poinz shows an error ?

<img width="141" alt="image" src="https://user-images.githubusercontent.com/1777143/173609910-99c07580-38be-428d-a01b-cb205d4c734e.png">

<img width="296" alt="image" src="https://user-images.githubusercontent.com/1777143/173610067-23edf322-969f-4eba-b99e-58cb6c7ef91b.png">


Usually there is an error if you have two cards with the same value 

<img width="297" alt="image" src="https://user-images.githubusercontent.com/1777143/173610269-ba9665d1-4b07-4cea-aa62-65d80d298bd9.png">

**Reply by jwoeckinger** [reply]
On a different PC on a different day everything works like expected. I have to go back to the original PC, but with your hints & the Log (i missed until now) i hope will solve the issue. Thanks for the quick and comprehensive reply! br Johann
